test/testObjects/value.json

```diff
@@ -1,6 +1,7 @@
 {
-  "root_path": "./",
-  "test/testObjects": [
+  "root_path": "./test",
+  "namespace": "Test\\TestObjects",
+  ".": [
```

Comment:
Das resultiert in der Generierung von `namespace Test\TestObjects\.;`, was syntaktisch nicht korrekt ist. Außerdem werden die Objekte ins falsche Verzeichnis generiert. Das scheint noch nicht zu funktionieren.

Reply:
Da hast du Recht das funktioniert noch nicht so wie gewünscht. Das bringe ich in Ordnung.

Reply:
Das habe ich korrigiert und es sollte jetzt funktionieren . und ./ sind sowohl für den root_path, als auch als Key für das Objekt array valide Werte die sich entsprechend verhalten.